Extension/src/LanguageServer/ui.ts

```diff
@@ -68,8 +68,6 @@ export class LanguageStatusUI {
     private intelliSenseStatusItem: vscode.LanguageStatusItem;
     private readonly updatingIntelliSenseText: string = localize("updating.intellisense.text", "IntelliSense: Updating");
     private readonly idleIntelliSenseText: string = localize("idle.intellisense.text", "IntelliSense: Ready");
-    private readonly missingIntelliSenseText: string = localize("absent.intellisense.text", "IntelliSense: Not configured");
-
     // Tag parse language status
```

Comment:
I think there's some more dead code in here after deleting that big block in `client.ts`.  `showConfigureCustomProviderMessage` and `showConfigureCompileCommandsMessage` should not be used anymore.  I also thought there was one about setting includePath too.